css/styles.css

```diff
@@ -0,0 +1,125 @@
+/*      high-level     */
+body {
```

Comment:
Setting styles on html elements like this is ok, but a word of caution: you may continue to build on top of this stylesheet, and may encounter a situation where your css styles seem to not apply or require far more styling than you would expect. That usually is a sign that a high level style is impacting other elements in how they are styled. However, they can also be a way to make updates to your design rapidly in the sense that you change your `h3` definition and now ALL the h3's in use are changed. CSS variables capture this idea and can be a good way to think about how to write maintainable (this won't be impossible to change or update in the future) code